core/src/banking_stage.rs

```diff
@@ -2145,6 +2146,7 @@ impl BankingStage {
                         &bank.feature_set,
                         bank.vote_only_bank(),
                         bank.as_ref(),
+                        bank.get_transaction_account_lock_limit(),
```

Comment:
The call here will prevent us from passing `SanitizedTransaction`s that cannot be locked to qos, which is called below in `process_transactions`